pandas/_libs/tslibs/ccalendar.pxd

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# cython: profile=False
```

Comment:
is this *only* used in fields? no-where else?

Reply:
Only implemented in fields.  Haven't updated others yet.  A couple places in offsets, a few more fastpaths in Timestamp.